CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.18)
+cmake_minimum_required(VERSION 3.20)
```

Comment:
why the change to cmake 3.20?

Reply:
Sorry; my bad! I thought CMake <=3.20 would complain with `SET CMP0116 OLD`. I'll change it back to 3.18.

Reply:
`if(POLICY` checks if cmake version supports this particular policy, but it doesn't error out if POLICY is not supported https://cmake.org/cmake/help/v3.0/manual/cmake-policies.7.html#introduction
`To manage policies without increasing the minimum required CMake version, the [if(POLICY)](https://cmake.org/cmake/help/v3.0/command/if.html#command:if) command may be used`